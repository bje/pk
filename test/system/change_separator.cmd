../../src/fieldx -S- 1 3 5
