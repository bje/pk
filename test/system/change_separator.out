A-C-E
a-c-e
